mlflow/server/js/src/utils/Utils.js

```diff
@@ -107,11 +111,40 @@ class Utils {
         res = <a href={url}>{res}</a>;
       }
       return res;
+    } else if (run.source_type === "NOTEBOOK") {
+      const notebookIdTag = 'mlflow.databricks.notebookID';
+      const webappUrlTag = 'mlflow.databricks.webappURL';
+      const notebookId = tags[notebookIdTag] && tags[notebookIdTag].value;
+      const webappUrl = tags[webappUrlTag] && tags[webappUrlTag].value;
+      if (notebookId && webappUrl) {
+        res = (<a href={`${webappUrl}/#notebook/${notebookId}`}>{run.source_name}</a>);
+      }
+      return res;
     } else {
       return res;
     }
   }
 
+  /**
+   * Returns an svg with some styling applied.
+   */
+  static renderSourceTypeIcon(sourceType) {
+    const imageStyle = {
+      height: '20px',
+      position: 'relative',
+      top: '-1px',
+      right: '3px',
+    };
+    if (sourceType === "NOTEBOOK") {
+      return <img style={imageStyle} src={notebookSvg} />;
+    } else if (sourceType === "LOCAL") {
+      return <img style={imageStyle} src={laptopSvg} />;
+    } else if (sourceType === "PROJECT") {
+      return <img style={imageStyle} src={laptopSvg} />;
```

Comment:
Why do these both have the laptop icon? I think the project icon should be a folder to differentiate the two. Otherwise most users will just see a laptop all the time.

Reply:
Sorry, it's changed in the latest commit. It should now be the folder icon.